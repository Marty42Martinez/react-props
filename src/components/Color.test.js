import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color component', () => {
  it('renders a Color', () => {
    const wrapper = shallow(<Color name="dark orchid" hex="#9932CC" rgb={{ red: 153, green: 50, blue: 204 }}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
