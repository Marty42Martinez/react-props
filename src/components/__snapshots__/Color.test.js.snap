// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color component renders a Color 1`] = `
<dl>
  <dt>
    Color Name
  </dt>
  <dd>
    dark orchid
  </dd>
  <dt>
    Hex
  </dt>
  <dd>
    #9932CC
  </dd>
  <dt>
    RGB
  </dt>
  <dd>
    Red-
    153
  </dd>
  <dd>
    Green-
    50
  </dd>
  <dd>
    Blue-
    204
  </dd>
</dl>
`;
